client/src/Clusters/TopCharts/SingleAppContainer.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import starIcon from './Images/starIcon.svg';
+
+class SingleAppContainer extends Component {
+  render() {
+    return (
```

Comment:
Optional/Super nit.

One cool thing you can do is before the return you can do 

```
const {listNum, title, category, url} = this.props;
```

Then in ur JSX you'll do 
```
<p class="num">{listNum}</p>
```

That way you dont need to type out this.props.listNum, this.props.title.

Reply:
This is called destructuring in case you wanted to read up on it: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment